socialhome/frontend/components/streams/Stream.vue

```diff
@@ -96,16 +103,47 @@ export default Vue.component("stream", {
                 () => this.$nextTick( // Wait for new content to be rendered
                     () => this.$scrollTo("body")))
         },
+        loadStream() {
+            const options = {params: {}}
+            const lastContentId = this.$store.state.contentIds[this.$store.state.contentIds.length - 1]
+            if (lastContentId && this.$store.state.contents[lastContentId]) {
+                options.params.lastId = this.$store.state.contents[lastContentId].through
+            }
+
+            switch (this.$store.state.stream.name) {
+                case "followed":
+                    this.$store.dispatch(streamStoreOperations.getFollowedStream, options)
+                    break
+                case "public":
+                    this.$store.dispatch(streamStoreOperations.getPublicStream, options)
+                    break
+                case "tag":
+                    options.params.name = this.tag
```

Comment:
There must be some kind of problem in passing in `this.tag` since locally I accessed `http://127.0.0.1:8000/streams/tag/foo/` and got an error notification (those are nice btw! :)). The API call is this: `http://127.0.0.1:8000/api/streams/tag//` so tag is empty.

Reply:
Problem spotted. Mistype.